examples/ibm-power/variables.tf

```diff
@@ -1,33 +1,94 @@
-// Service / Account
+# Service // Account
 variable "ibm_cloud_api_key" {
   description = "API Key"
   type        = string
   default     = "<key>"
 }
 variable "region" {
-  description = "Reigon of Service"
+  description = "Region of Service"
   type        = string
   default     = "<e.g dal>"
 }
 variable "zone" {
   description = "Zone of Service"
   type        = string
-  default     = "<e.g 12>"
+  default     = "<e.g dal12>"
```

Comment:
variables have defaults that work.  If you want to force the user to provide the value do not specify the default.  It you want it to work without supplying a value use a string like "dal12"

Reply:
I think default is fine because it shows the user what kind of information goes into that field. Without giving an example in the default field, it might be confusing to say what kind of inputs are accepted.